### ISSUE TITLE
Example in readme does not work with imagemin v7

### ISSUE THREAD

**Issue by simonxabris** [imagemin/imagemin-mozjpeg#45]
Hi

With the [v7 release](https://github.com/imagemin/imagemin/releases/tag/v7.0.0) Imagemin introduced some breaking changes to the public API which means the example code in the readme no longer works.

I think the example in the readme should be updated so that it works with the latest major imagemin version or provide two examples, one for v7 and one for v6. If you think this is a good idea I'd be happy to submit a PR!

**Comment by notiv-nt** [comment]
```javascript
const imagemin = require('imagemin');
const imageminMozjpeg = require('imagemin-mozjpeg');

(async () => {
  await imagemin(['images/*.jpg'], {
    destination: 'build',
    plugins: [
      imageminMozjpeg()
    ]
  });

  console.log('Images optimized');
})();
```

**Comment by 1000ch** [comment]
Solved by 9f3f702b44a85e546c57ed0e1044fade757abd00.
